src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -186,14 +197,97 @@ export class EventStoreComponent implements OnInit, OnDestroy {
       }
 
       return combinedTimelineData;
+  }  
+
+  private addKindToEvents(events: DataSet<DataItem>, consumedEvents: DataSet<DataItem>, parsedEvents: DataSet<DataItem>): DataSet<DataItem> {
+    /*
+        Consumed events (the ones returned by consume of timelineGenerator) do not have a kind attached to them - this function adds the kind
+        to each object so that events can be compared with the IConcurrentEventsConfig objects.
+    */
+    consumedEvents.items.forEach(consumedEvent => {
+        let idx = 0;
+        for (let event of events) {
+            if (consumedEvent.id.indexOf(event.eventInstanceId) == 0) {
+                break;
+            }
+            idx++;
+        }
+        let removedEvent = events[idx];
+        events.splice(idx, 1);
+        consumedEvent['kind'] = removedEvent.kind;                  
+        parsedEvents.add(consumedEvent);
+    });
+  }
+
+  private getConcurrentEventsData(): DataSet<DataItem> {
```

Comment:
is this just for testing?

Reply:
Yep, just to test sample application events